packages/react-query/package.json

```
@@ -42,7 +42,7 @@
     "src"
   ],
   "bin": {
-    "suspensive-react-query-switch": "dist/scripts/suspensive-react-query-switch.cjs"
+    "suspensive-react-query": "dist/scripts/cli.cjs"
```

How about providing 2 way? in cli world, kind of shorthand is usual.

```suggestion
    "suspensive-react-query": "dist/scripts/cli.cjs",
    "srq": "dist/scripts/cli.cjs"
```

---

I agree, I completely forgot about that!